### ISSUE TITLE
why the eval result is  not  good?

### ISSUE THREAD

**Issue by 15241192958** [google/automl#315]
![image](https://user-images.githubusercontent.com/56427396/80301771-aa6a4b80-87d8-11ea-815b-37d77e968115.png)
i finetuned 6000 own images with efficiendetd1 (in coco ckpt ),  after 3hours ，the loss is 0.8,the AP50 is  high ,but the accuracy is low  0.45,而且他会有很多误检(my Poor english)，what should i do to improve its AP?

**Comment by qtw1998** [comment]
> ![image](https://user-images.githubusercontent.com/56427396/80301771-aa6a4b80-87d8-11ea-815b-37d77e968115.png)
> i finetuned 6000 own images with efficiendetd1 (in coco ckpt ), after 3hours ，the loss is 0.8,the AP50 is high ,but the accuracy is low 0.45,而且他会有很多误检(my Poor english)，what should i do to improve its AP?

尝试改下anchor size

**Comment by elesun2018** [comment]
when i run eval code on coco with pretrained models, meet a problem as follows:
automl0401/efficientdet# python main.py --mode=eval --model_name=${MODEL} --model_dir=${CKPT_PATH} --validation_file_pattern=tfrecord/val* --val_json_file=annotations/instances_val2017.json --hparams="use_bfloat16=false" --use_tpu=False

CKPT_PATH is the pretrained d0 model.

DONE (t=34.86s).
Average Precision (AP) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.002
Average Precision (AP) @[ IoU=0.50 | area= all | maxDets=100 ] = 0.005
Average Precision (AP) @[ IoU=0.75 | area= all | maxDets=100 ] = 0.002
Average Precision (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.003
Average Precision (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.001
Average Precision (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 1 ] = 0.006
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 10 ] = 0.011
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.012
Average Recall (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.018
Average Recall (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.006
Average Recall (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.004
INFO:tensorflow:Finished evaluation at 2020-04-28-17:12:04
INFO:tensorflow:Finished evaluation at 2020-04-28-17:12:04
I0428 17:12:04.599111 140494972888896 evaluation.py:275] Finished evaluation at 2020-04-28-17:12:04
INFO:tensorflow:Saving dict for global step 0: AP = 0.0022378154, AP50 = 0.0052162167, AP75 = 0.0015809378, APl = 0.00015473206, APm = 0.0014778649, APs = 0.0029822278, ARl = 0.0036031725, ARm = 0.006258475, ARmax1 = 0.005900326, ARmax10 = 0.011014862, ARmax100 = 0.011780873, ARs = 0.018089531, box_loss = 0.0, cls_loss = 0.02997985, global_step = 0, loss = 0.11981841
I0428 17:12:04.600037 140494972888896 estimator.py:2049] Saving dict for global step 0: AP = 0.0022378154, AP50 = 0.0052162167, AP75 = 0.0015809378, APl = 0.00015473206, APm = 0.0014778649, APs = 0.0029822278, ARl = 0.0036031725, ARm = 0.006258475, ARmax1 = 0.005900326, ARmax10 = 0.011014862, ARmax100 = 0.011780873, ARs = 0.018089531, box_loss = 0.0, cls_loss = 0.02997985, global_step = 0, loss = 0.11981841
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 0: models/pretrained_models/efficientdet-d1/model

such low AP values ? why ?
thanks.

**Comment by zishanahmed08** [comment]
@elesun2018 Did u figure it out?
